quad_utils/src/rviz_interface.cpp

```diff
@@ -531,7 +532,7 @@ void RVizInterface::robotStateCallback(
     state_estimate_trace_pub_.publish(state_estimate_trace_msg_);
 
   } else if (pub_id == GROUND_TRUTH) {
-    transformStamped.child_frame_id = "/ground_truth/body";
+    transformStamped.child_frame_id = tf_prefix_ + "/body";
```

Comment:
Is there a reason why this one just has tf_prefix_ + "/body" whereas the others have estimate and trajectory? I think we would want to leave "ground_truth" in there.

Reply:
The visualization.launch has a ground_truth namespace already.

Reply:
This has been resolved in the most recent push.